vault/util_test.go

```diff
@@ -3,19 +3,40 @@
 
 package vault
 
-import "testing"
+import (
+	"math/rand"
+	"testing"
+)
+
+const n = 1024
 
 func TestMemZero(t *testing.T) {
-	b := []byte{1, 2, 3, 4}
+	r := rand.Intn(n)
+	b := randbytes(r)
 	memzero(b)
```

Comment:
TBH, given the rather extensive disclaimer around this function:

https://github.com/openbao/openbao/blob/88945c0791e71caa61883ba0851e856e4955c076/vault/util.go#L11-L29

I wonder if it is better to simply remove it? memguard is probably preferable: https://github.com/awnumar/memguard -- but it'd be great if this was a native part of the Go toolchain.

Reply:
We may discuss your proposal (or using https://github.com/awnumar/memcall for storing key material?), but this PR is about tests only :-)

Reply:
@DanGhita You might also have opinions about removing this. :-) for the future, though!